README.md

```diff
@@ -8,25 +8,27 @@ Here we define the Gatling chart with its templates and default configuration.
 ## Required values
 
 In order to run Gatling, it must be provided with a simulation file which defines the
-test(s) to run. This configuration can be provided either as a `configMap`, or as a URL
+test(s) to run. This configuration can be provided either as a `configmap`, or as a URL
```

Comment:
totally random nitpick: why not use the official capitalization since you already change it?

Reply:
Changed it, hope this is what you were thinking about! :)